styles/style.css

```diff
@@ -0,0 +1,327 @@
+* {
+  box-sizing: border-box;
+  margin: auto;
+}
+main {
+  padding: 0.5rem 9.5rem 4rem 9.5rem;
+}
+img {
+  max-width: 100%;
+}
+ul {
+  list-style: none;
+}
+a:link {
+  text-decoration: none;
+}
+/* header style start */
+header > div {
+  background-image: url('../images/header-image.jpg');
+  display: grid;
+  padding-left: 5rem;
+  padding-right: 5rem;
+  padding-top: 2rem;
+  padding-bottom: 6rem;
+  grid-template-rows: 1fr 1fr 1fr 1fr;
+  grid-template-columns: 36% 1fr 25%;
+}
+header img[src="images/adoptadog-logo.svg"] {
+  max-width:100%;
+}
+header img[src="images/cart-icon.svg"] {
+  width: 35px;
+  padding-right: 0.4rem;
+}
+header .cart-icon-money {
+  display: flex;
+  flex-direction: row;
+}
+header p {
+  color: white;
+  font-size: 0.8rem;
+}
+header>div div:first-child {
+  grid-area: 1 / 1 / 5 / 2;
+}
+header div h1 {
+  color: #81B741;
+  font-size: 1.2rem;
+}
+header>div div:nth-child(2) {
+  grid-area: 1 / 3 / 2 / 4;
+}
+header>div button {
+  grid-area: 2 / 3 / 3 / 4;
+  background-color: #81B741;
+  border: none;
+  width: 5rem;
+  color: white;
+  width: 7rem;
+  height: 1.5rem;
+}
+/* header style end */
+/* nav style start */
+header nav {
+  background-color: #EBEBEB;
+  padding-left: 9.5rem;
+}
+header nav ul li {
+  display: inline-flex;
+}
+header nav ul {
+  padding: 0;
+}
+header nav ul li:not(:last-child)::after {
+  content: '/';
+  padding: 0.9rem;
+}
+/* nav style end */
+/* footer style start */
+footer {
+  background-color: #037EAA;
+  padding: 2.5rem;
+}
+footer ul {
+  display: table;
+  text-align: center;
+  padding-top: 1rem;
+  padding-bottom: 1.5rem;
+}
+
+footer img {
+  max-width: 30%;
+  display: block;
+  padding-bottom: 1.2rem;
+}
+footer ul li {
+  display: inline-flex;
+}
+footer ul li a {
+  color: white;
+}
+footer ul li:not(:last-child)::after {
+  content: '/';
+  color: white;
+  padding: 0.5rem;
+}
+footer p {
+  color: white;
+  text-align: center;
+}
+/* footer style end */
+/* dog-card style start */
+.container-dog-card {
+  border: 2px solid #EBEBEB;
+  padding-bottom: 0.7rem;
+}
+.button-dog-card {
+  background-color: #81B741;
+  color: white;
+  font-weight: bold;
+  width: 90%;
+  border-radius: 2px;
+  border: none;
+  padding: 0.4rem 3rem 0.4rem 3rem;
+  display: flex;
```

Comment:
you shoulnd't need flex to layout this button. But you will need to center the text

Reply:
Uhh, without the flex, I do not know how to center the button. Is there any other way can center them?